docs/installing.rst

```diff
@@ -11,25 +11,27 @@ Installing the Atomic Developer Bundle
 1. Install a Virtualization Provider
 ------------------------------------
 
-Two virtualization providers have been tested with the ADB.
+Two virtualization providers have been tested with the ADB; VirtualBox and Libvirt.
+
+The following virtualization providers are suggested for the respective operating systems:
 
 * Microsoft Windows and Mac OS X
 
-  The suggested virtualization provider is `VirtualBox`_. Installation
+  The suggested virtualization provider for Microsoft Windows and Mac OS X is `VirtualBox`_. Installation
   instructions are available `online`_. While the latest stable shipping release
   should work, the majority of testing has been done with version 5.0.0 on Mac
   OS X and 5.0.8 on Microsoft Windows.
 
 .. _VirtualBox: https://www.virtualbox.org
-.. _oneline: https://www.virtualbox.org/manual/UserManual.html
+.. _online: https://www.virtualbox.org/manual/UserManual.html
 
 * Fedora
 
-  Two different virtualization providers are supported on Linux, `VirtualBox`_
+  Two different virtualization providers are supported on Linux; `VirtualBox`_
   and `libvirt <http://libvirt.org/>`_. The choice as to which to use should be
   driven by your preferences and environmental concerns and is outside of the
   scope of this document. Both will work equally well in their default
-  configuration. You may wish to read the section on file synchronization when
+  configuration. You may wish to read the section on `file synchronization`_ when
   making this decision.
 
   * VirtualBox installation instructions are available `online at the VirtualBox
```

Comment:
Ugh, we need directions here.  This is the fedora section, so we should be able to advise using the distribution build via:

`sudo dnf install VirtualBox-4.3`